hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/InternalOperationService.java

```diff
@@ -35,6 +35,11 @@
  */
 public interface InternalOperationService extends OperationService, PacketHandler {
 
+    /**
+     * Returns the size of the response queue.
+     *
+     * @return the size of the response queue.
```

Comment:
It is invalid to have a `@return` tag with no content, but it's fine not to have the tag at all. AFAIK.